### ISSUE TITLE
Can't merge from GitButler: "Pull request is not next in stack

### ISSUE THREAD

**Issue by ipsquiggle** [gitbutlerapp/gitbutler#5726]
### Version

0.14.1

### Operating System

Windows

### Distribution Method

msi (Windows)

### Describe the issue

As per the screenshot, I'm unable to merge from the gitbutler UI. The button is greyed out, claiming that this PR is not next in the stack.

However, this is is the ONLY PR so I really don't know what I can do to fix it.

![image](https://github.com/user-attachments/assets/7261927f-ae54-41fb-b890-136834bedd11)
![image](https://github.com/user-attachments/assets/407ed508-d1ba-443b-bb64-17251e867c05)

### How to reproduce

Unsure how I arrived in this state, but this is persistent, I have had to merge multiple PRs through the GitHub UI, as it no longer seems to work in GitButler at all.

### Expected behavior

a) Obviously, it would make sense if an unblocked PR can be merged through the UI
b) In case this was genuine, I still wouldn't know what was blocking it, so some kind of indication as to what the blocking PR or branch was would be very useful!

**Comment by Byron** [comment]
Thanks a lot for reporting, and sorry for the hassle!

It does seem unintuitive, and maybe it's just a UI hickup. CC @ndom91 .

**Comment by SnaveSutit** [comment]
I'm also having this issue on 0.14.6. Should I open a new issue for it?

![Image](https://github.com/user-attachments/assets/3559589c-ed7d-4f1e-9961-ee310e31d87a)

**Comment by krlvi** [comment]
Hey! Thank you for reporting this. I am wondering - is it possible that the pull request's target is not the main branch in the repo? Could you go to the pull request and check that this looks correct?
<img width="1089" alt="Image" src="https://github.com/user-attachments/assets/48becead-2299-4563-af03-e02409dc1734" />

Would it be possible to also verify/ check if this config is set in the repo? https://docs.gitbutler.com/features/stacked-branches#automatic-branch-deletion

**Comment by SnaveSutit** [comment]
Definitely targeting the main branch on both

![Image](https://github.com/user-attachments/assets/234816de-9f1b-4da7-8e3f-405331cc82a1)

![Image](https://github.com/user-attachments/assets/34838152-43d1-49c8-9c75-ea36681d1a82)

**Comment by smitmartijn** [comment]
I've recently started having this same issue on `Version 0.18.2 (0.18.2)` - and it only started after I updated the remote of the git repository. I renamed the github repo name and used the CLI to change the remote.

It wasn't until I removed the project from Gitbutler and added it back again that it started allowing me to merge from GB again. 

Maybe this note helps someone who's running into the same issue.

**Comment by kindkitchen** [comment]
> I've recently started having this same issue on `Version 0.18.2 (0.18.2)` - and it only started after I updated the remote of the git repository. I renamed the github repo name and used the CLI to change the remote.
> 
> It wasn't until I removed the project from Gitbutler and added it back again that it started allowing me to merge from GB again.
> 
> Maybe this note helps someone who's running into the same issue.

I also confirm, that faced with this issue after renaming my repository.

**Comment by kindkitchen** [comment]
## I was able to reproduce and fix this issue

#### Intro
- on github i had repository `some-example`
- locally - had remote `origin` which will point to `.../some-example`
- **i rename github's repository** to `demo`
- locally nothing were changed and btw continue to work

#### Issue
Yes after this I had problem `Pull request is not next in stack` _(so i was enforced to complete my prs manually)_

#### Solution
**I also rename my remote `origin`** and it helped!

p.s.
```sh
# command to rename remote url
git remote -v
git remote set-url origin https://github.com/kindkitchen/demo.git
```
